src/core/http/data_loader.rs

```diff
@@ -65,6 +65,7 @@ impl Loader<DataLoaderRequest> for HttpDataLoader {
     async fn load(
         &self,
         keys: &[DataLoaderRequest],
+        http_filter: HttpFilter,
```

Comment:
Store the filter in `HttpDataLoader` as a field

Reply:
Can you please explain more. Where is it used later in the code?